internal/manifest/report.go

```diff
@@ -18,6 +19,8 @@ type ReportEntry struct {
 	Slices      map[*setup.Slice]bool
 	Link        string
 	FinalSHA256 string
+	// If HardLinkID is greater than 0, all entries with the same id represent hard links to the same inode.
+	HardLinkID uint64
```

Comment:
Instead of `HardLinkID/hard_link_id`, I suggest naming this `Inode/inode`. More general idea, it's naturally an integer, and reflects the semantics in the filesystem.

Reply:
Isn't inode going to suggest there is some relationship with the physical inode in the filesystem?

Reply:
Discussed offline and agreed that this is a hint that this is an inode, it could in fact be the physical one.